src/main/java/org/spongepowered/common/config/category/TimingsCategory.java

```diff
@@ -33,22 +33,22 @@
 @ConfigSerializable
 public class TimingsCategory extends ConfigCategory {
 
-    @Setting
+    @Setting(comment = "???")
```

Comment:
??? no clue i looked briefly and it seems to end in https://github.com/SpongePowered/SpongeCommon/blob/6077a35adcb7fd95d4886ffa5aa408a63ac1b4f2/src/main/java/org/spongepowered/common/relocate/co/aikar/timings/TimingHandler.java#L72

not sure what the point is